plugins/@grouparoo/hubspot/__tests__/integration/hubspot-export.ts

```diff
@@ -47,7 +48,7 @@ async function deleteUsers() {
         await client.deleteContact(contact.vid);
       }
     } catch (error) {
-      if (!error.toString().match(/Request failed with status code 404/)) {
+      if (!error.toString().match(/Request failed with status code 400/)) {
```

Comment:
This is in a test so it's probably fine, but why did this change? I believe the intention here was to ignore any issues with deleting non-existent users.

Reply:
Good catch. I think it's because I was not running nock correctly and I got some bad errors and tried to fix it this way but needs to go back.